core/templates/dev/head/forms/formBuilder.js

```diff
@@ -1163,7 +1163,10 @@ oppia.directive('schemaBasedEditor', [function() {
       onInputFocus: '='
     },
     templateUrl: 'schemaBasedEditor/master',
-    restrict: 'E'
+    restrict: 'E',
+    controller: function() {
+      document.execCommand('enableObjectResizing', false, false);
```

Comment:
(Also, just to check, could you please test in a couple of other browsers to ensure there are no side effects?)